src/installer/destinations.py

```diff
@@ -176,20 +208,32 @@ def write_script(
         with io.BytesIO(data) as stream:
             entry = self.write_to_fs(Scheme("scripts"), script_name, stream)
 
-            path = os.path.join(self.scheme_dict[Scheme("scripts")], script_name)
+            path = self._path_with_destdir(Scheme("scripts"), script_name)
             mode = os.stat(path).st_mode
             mode |= (mode & 0o444) >> 2
             os.chmod(path, mode)
 
             return entry
 
+    def _compile_bytecode(self, scheme: Scheme, record: RecordEntry) -> None:
+        """Compile bytecode for a single .py file."""
+        if scheme not in ("purelib", "platlib"):
+            return
+
+        target_path = self._path_with_destdir(scheme, record.path)
+        for level in self.bytecode_optimization_levels:
+            kwargs: Dict[str, Any] = {}
+            if sys.version_info >= (3, 9):  # pragma: no cover
+                kwargs["stripdir"] = str(self.destdir)
+            compileall.compile_file(target_path, optimize=level, **kwargs)
```

Comment:
Non-blocking comment: perhaps you want to pass `quiet=1` here? It feels a bit weird that the install itself is entirely quiet, yet compileall is very verbose.

Reply:
Good point, yes, I think `quiet=1` (printing only on errors) is appropriate.